### ISSUE TITLE
iptables panic leading to CI test failures

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#25504]
https://storage.googleapis.com/istio-prow/logs/integ-distroless-k8s-tests_istio_postsubmit/1792/artifacts/security-1561ecc9de344180b05696/TestReachability/reachability-1-80906-state952705050/vm-v1-d4468d46d-9nwsj-istio-proxy.log
https://storage.googleapis.com/istio-prow/logs/integ-distroless-k8s-tests_istio_postsubmit/1791/artifacts/security-4cf7a47365f5425d8bcde4/TestReachability/reachability-1-20457-state367595562/vm-v1-795d595484-hlbnk-istio-proxy.log

**Comment by lambdai** [comment]
```
iptables-restore --noflush /tmp/iptables-rules-1594751504941469528.txt993273918
Writing following contents to rules file:  /tmp/ip6tables-rules-1594751504945812290.txt096504453
* nat
-N ISTIO_REDIRECT
-A ISTIO_REDIRECT -p udp -j REDIRECT --to-ports 15013
COMMIT

ip6tables-restore --noflush /tmp/ip6tables-rules-1594751504945812290.txt096504453
ip6tables-restore v1.6.1: ip6tables-restore: unable to initialize table 'nat'
```
Somehow the NAT tables is not created as expected.

**Comment by howardjohn** [comment]
@lambdai the  test failed 2x in a row with no apparent related configs. Maybe the actual nodes the test runs on changed?

**Comment by lambdai** [comment]
I have the same feeling: It's likely the node/docker env causing it

This 4 liner ip6tables rules are absolute reasonable. My machine could restore the 4 liner or 2 rules perfectly.

**Comment by howardjohn** [comment]
https://prow.istio.io/view/gcs/istio-prow/logs/integ-distroless-k8s-tests_istio_postsubmit/1794 passed after this one. Lets keep an eye on this and see if any pattern comes up

**Comment by howardjohn** [comment]
fixed by #25590 I think
